scripts/yocto-build-and-test.sh

```diff
@@ -124,14 +125,18 @@ FILESEXTRAPATHS_prepend_pn-mender-binary-delta := "$WORKSPACE/mender-binary-delt
 PREFERRED_VERSION_pn-mender-binary-delta = "$mender_binary_delta_version"
 EOF
 
-    local mender_monitor_version=$(tar -xf $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz ./mender-monitor/.version -O | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
+    local mender_monitor_version=$(tar -Oxf $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz ./mender-monitor/.version | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
+    for i in $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz; do
+     tar -C $WORKSPACE/stage-artifacts -xvzf $i
+    done
     if [ -z "$mender_monitor_version" ]; then
         mender_monitor_version="master-git"
     fi
     cat >> $BUILDDIR/conf/local.conf <<EOF
 LICENSE_FLAGS_WHITELIST += "commercial_mender-monitor"
-SRC_URI_pn-mender-monitor = "file:///$WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz"
-PREFERRED_VERSION_pn-mender-monitor = "$mender_monitor_version"
+FILESEXTRAPATHS_prepend_pn-mender-monitor := "/$WORKSPACE/stage-artifacts/:"
+SRC_URI_pn-mender-monitor = "file://mender-monitor/"
+PREFERRED_VERSION_pn-mender-monitor = "master"
```

Comment:
I think hardcoded "master" is wrong. Leftover from debugging?

In fact `PREFERRED_VERSION` probably has no effect, since there is only one recipe for this package. Just leave it like it was, for the future when we have multiple recipes.

Reply:
fixed.